cloud/pkg/router/provider/rest/rest.go

```diff
@@ -201,7 +201,13 @@ func (r *Rest) GoToTarget(data map[string]interface{}, stop chan struct{}) (inte
 	if !ok || len(content) == 0 {
 		return nil, errors.New("invalid convert to []byte")
 	}
-	req, err := httpUtils.BuildRequest(http.MethodPost, r.Endpoint, bytes.NewReader(content), "", "")
+	nodeName, ok := data["nodeName"].(string)
+	if !ok {
+		err := fmt.Errorf("input data does exist valid value \"nodeName\"")
```

Comment:
For compatibility reasons, it is fine to modify this.
But the log should be `input data does not exist valid value \"nodeName\"`?

Reply:
ok